content/cloud/noops-sql-db.md

```diff
@@ -69,34 +67,41 @@ name = "c3"
 sqlite_databases = ["default"]
 ```
 
-The database will be created when the Spin application is deployed. The data persists Spin application invocations and updates. 
+The above configuration will last the lifespan of the application.
 
-```bash
-$ spin cloud deploy
-Deploying app ..
-Creating database inspirational-pig (default) for app "quickstart"
-```
+### Preparing Fermyon Cloud Schema
 
-You can check to see your database by using the `spin cloud sqlite` command:
+Bootstrapping your Cloud SQLite database schema is done via the `spin cloud sqlite execute` subcommand. Your NoOps SQLite application on localhost will have already assigned a name for your database. All we need to do is obtain the name by typing the following command (in your local NoOps SQLite application's directory):
```

Comment:
Maybe we need to start this with more context.  This might help address the evident confusion about identifying the local and cloud databases.  E.g.:

When Fermyon Cloud creates a SQLite database, the database is empty. To be useful, you'll need to create tables, and potentially populate some of those tables with initial data.  You can do this either from within the application, or from the command line using the `spin cloud sqlite execute` command.

> When running locally, you can set up the database using `spin up --sqlite`, but this option isn't available when deploying to Fermyon Cloud.

To set up the cloud database from within your application...

To set up the cloud database from the command line...

(these latter two might need to be another level of subheading, in which case it might be worth busting this topic up a level so we're not dealing with four levels of hierarchy!)

Reply:
I see there was a "from within your application" section in an earlier draft which was removed at Ryan's guidance, so please ignore it in the above suggestion!